client/src/localization/languages/Zh.tsx

```diff
@@ -120,64 +144,100 @@ export default {
   com_endpoint_max_output_tokens: '最大输出词元数',
   com_endpoint_openai_temp:
     '值越高表示输出越随机，值越低表示输出越确定。建议不要同时改变此值和Top P。',
-  com_endpoint_openai_max: '最大生成词元数。输入词元长度由模型的上下文长度决定。',
+  com_endpoint_openai_max: 
+    '最大生成词元数。输入词元长度由模型的上下文长度决定。',
   com_endpoint_openai_topp:
     '相较于随机性的另一个取样方法，称为核采样，模型选取输出词元中大于P值（概率密度在整个概率分布中的比例）的结果。比如 top_p=0.1 表示只有概率占比为前10%的词元才会被考虑作为输出。建议不要同时改变此值和随机性。',
   com_endpoint_openai_freq:
     '值介于-2.0到2.0之间。正值表示根据已有词元的频率惩罚重复词元结果, 从而减少模型输出重复词。',
   com_endpoint_openai_pres:
-    '值介于-2.0到2.0之间。正值表示根据出现词元惩罚重复词元结果, 从而增加模型提出新主题的可能性。',
+    '值介于-2.0到2.0之间。正值表示根据出现词元惩罚重复词元结果, 从而增加模型提出新话题的可能性。',
   com_endpoint_openai_custom_name_placeholder: '为ChatGPT设置一个名称',
-  com_endpoint_openai_prompt_prefix_placeholder: '可以在系统消息中设置自定义指令，默认为空',
-  com_endpoint_frequency_penalty: '频度惩罚',
-  com_endpoint_presence_penalty: '出现惩罚',
+  com_endpoint_openai_prompt_prefix_placeholder: 
+    '在消息开头添加系统级提示词，默认为空',
+  com_endpoint_anthropic_temp:
+    '值介于0到1之间。 对于分析性/选择性任务，值应更接近0；对于创造性和生成性任务，值应更接近1。我们建议更改该参数或Top-p，但不要同时更改这两个参数。',
+  com_endpoint_anthropic_topp:
+    'Top-p（核采样）会改变模型选择输出词元的方式。从概率最大的K（参见topK参数）向最小的K选择，直到它们的概率之和等于top-p值。',
+  com_endpoint_anthropic_topk:
+    'Top-k 会改变模型选择输出词元的方式。top-k为1意味着所选词是模型词汇中概率最大的（也称为贪心解码），而top-k为3意味着下一个词是从3个概率最大的词中选出的（使用随机性）。',
+  com_endpoint_anthropic_maxoutputtokens:
+    '响应中可生成的最大词元数。指定较低的值会得到更短的响应，而指定较高的值则会得到更长的响应。',
+  com_endpoint_anthropic_custom_name_placeholder: '为Anthropic设置一个名称',
+  com_endpoint_frequency_penalty: '重复惩罚度',
+  com_endpoint_presence_penalty: '话题新鲜度',
   com_endpoint_plug_use_functions: '使用函数',
   com_endpoint_plug_skip_completion: '跳过补全',
   com_endpoint_disabled_with_tools: '系统禁用',
   com_endpoint_disabled_with_tools_placeholder: '系统禁用',
   com_endpoint_plug_set_custom_instructions_for_gpt_placeholder:
-    '可以在系统消息中设置自定义指令，默认为空',
-  com_endpoint_set_custom_name: '设置一个自定义名，以方便寻找预设',
+    '在消息开头添加系统级提示词，默认为空',
+  com_endpoint_import: '导入',
+  com_endpoint_set_custom_name: '设置一个自定义名，以便您检索此预设',
+  com_endpoint_preset: '预设',
+  com_endpoint_presets: '预设',
   com_endpoint_preset_name: '预设名',
-  com_endpoint_presets_clear_warning:'确定要清除所有预设吗？这是不可逆转的。',
-  com_endpoint_config_key: '设置API Key',
-  com_endpoint_config_key_for: '设置API Key: ',
-  com_endpoint_config_key_encryption: '您的密钥将被加密并删除于:',
-  com_endpoint_config_key_import_json_key: '导入Service Account JSON Key',
-  com_endpoint_config_key_google_need_to: '你需要开启:',
-  com_endpoint_config_key_google_vertex_ai: 'Vertex AI API',
-  com_endpoint_config_key_google_vertex_api: '在Google云上(Google Cloud)，然后',
-  com_endpoint_config_key_google_service_account: '创建Service Account',
-  com_endpoint_config_key_google_vertex_api_role: '确保单击“创建并继续”以至少授予“Vertex AI 用户”角色。 最后，创建一个要在此处导入的 JSON 密钥。',
-  com_endpoint: '端口',
+  com_endpoint_new_topic: '新主题',
+  com_endpoint: '渠道',
   com_endpoint_hide: '隐藏',
   com_endpoint_show: '显示',
-  com_endpoint_examples: ' 例子',
+  com_endpoint_examples: ' 预设',
```

Comment:
Could you please elaborate more the reason why translate the `examples` to `presets`?

Reply:
nvm, the variable name is a bit confusing here, the actual content is `preset`.